### DISCUSSION TITLE
Adding Timeout support to the generated C source

### DISCUSSION THREAD

**Discussion by manojh93** [cantools/cantools#304] [Q&A]
May I please get some suggestions to add timeout support to the generated C source? Is it feasible? in case yes, what would be a better way of doing it?

**Comment by PikesterUK** [comment]
The GenSigTimeoutTime attribute could be used to determine what the timeout value should be. It would be easy to add a define for each message if that attribute is defined. 


Actual implimentation of a timeout would seem to be a layer of abstraction above what the C source generator does.
